### ISSUE TITLE
Use accessibilityLabel when available in NIAttributedLabel

### ISSUE THREAD

**Issue by lisafiedler** [jverkoey/nimbus#664]
I would like to be able to set the accessibilityLabel for in an NIAttributedLabel in some cases, but the accessibilityLabel is always set to the actual attributedText.string so I'm unable to set it myself. It seems like it should only be the attributedText if no accessibilityLabel is available.
